### PR TITLE
Added mobile menu

### PR DESCRIPTION
**Pull request by LeslieAine**
In this branch, I add an overlay for the mobile menu and js code to interact with the page

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+navHamIcon.addEventListener('click', () => {
+  mobileMenuOverlay.style.left = '0%';
+  mobileMenuOverlay.style.display = 'flex';
+});
+
+crossButton.addEventListener('click', () => {
+  hideMobileMenu();
+});
+
+mobileMenuLinks.forEach((mobileMenuLink) => {
```
Comment (brainconnect93) [inline script.js:21]: - [ ] Having made a great effort working on the mobile menu 👏 👏, I noticed some strange functionality in your work, when the `hamburger` icon is clicked the mobile menu should pop out over the `main page` but in your case, the mobile menu is coming out by the side of the `main page` and the `(X)` icon is not working for closing the mobile menu.

  - Kindly check the video below to see the aspect of the project am talking about 👇 

[screen-capture (2).webm](https://user-images.githubusercontent.com/78634841/210308427-b10e78f8-4abf-46f3-8d57-99a4207aff0c.webm)

> N.B :  annie

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :recycle:

### Hi @LeslieAine , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -35,6 +35,48 @@ html {
...
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  background-color: #3c3a39;
+  padding-top: 18px;
+  transition: all 0.5s ease-out;
+  overflow-x: hidden;
+}
```
Comment (mikethreels) [inline styles.css:48]: - [ ] good job fixing the issue the previous CR notified you about, however, when fixing this another bug was created. when loading the application the first time the mobile menu is open even in desktop mode, you can fix this by changing `display: flex;` to `display: none;` in the `mobile-menu-overlay` class. please make sure to check for any other bugs after making this change as well.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Leslie, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- linters are passing

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @LeslieAine , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVE :tada: 

## Highlights
- [ ] All linters are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**